chpl/chpl-service/src/main/resources/environment.properties.template

```diff
@@ -5,7 +5,8 @@ persistenceUnitName=openchpl
 springfox.documentation.swagger.v2.path=/api-docs
 basePath=/rest
 questionableActivityThresholdDays=0
-maxActivityRangeInDays=60
+#display jobs that are currently running or have completed within the last jobThresholdDays days
+jobThresholdDays=7 
```

Comment:
Can you / did you add this to the deployment files on Bamboo?

Reply:
I did not. I will!